frontend/src/pages/Home.tsx

```diff
@@ -1,11 +1,14 @@
 import { IonContent, IonFab, IonFabButton, IonFabList, IonIcon, IonPage } from "@ionic/react";
 import { analytics, menu, person } from "ionicons/icons";
 import LeafletMapContainer from "../components/LeafletMapContainer";
+import RoutePlanner from "../components/RoutePlanner";
 import "./Home.css";
 
+
 const Home: React.FC = () => (
   <IonPage>
     <IonContent fullscreen>
+      <RoutePlanner />
       <LeafletMapContainer />
```

Comment:
I would rather modify the E2E Tests than keeping a component that we no longer need at this location. I can take a look at this later today, shouldn't be too much of a change.